### PR TITLE
Extend max liquidity

### PR DESCRIPTION
**Pull request by poliwop**

### CODE REVIEW

**Comment by poliwop** [comment]
Task linked: [CU-86bwe2kum fix max_liquidity](https://app.clickup.com/t/86bwe2kum)

File: hydradx/model/amm/arbitrage_agent.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 
 # note that this function mutates state, test_agent, test_cex, and max_liquidity
-def process_next_swap(state, test_agent, test_cex, tkn_pair, ob_tkn_pair, buffer, max_liquidity, iters):
+def process_next_swap(state, test_agent, test_cex, tkn_pair, ob_tkn_pair, buffer, max_liquidity_dex, max_liquidity_cex, iters):
```
Comment (jepidoptera) [inline hydradx/model/amm/arbitrage_agent.py:6]: Minor quibble, but using 'test_dex' as the variable name here would seem more consistent.

**Review by jepidoptera — APPROVED** [review]
Looks correct, I left a couple of comments, but I don't think they should hold up the merge.

File: hydradx/model/amm/arbitrage_agent.py [hunk 2]
```diff
@@ -136,11 +134,23 @@ def get_arb_opps(op_state, cex_dict, config):
     return arb_opps
 
 
+def does_max_liquidity_allow_trade(dex_tkn_pair, cex_tkn_pair, max_liquidity_dex, max_liquidity_cex):
```
[the remark this answers is not in the document]
Comment (poliwop) [reply hydradx/model/amm/arbitrage_agent.py:137]: This makes sense, I'll fix it.

File: hydradx/model/amm/arbitrage_agent.py (hunk 1 above)
Comment (poliwop) [reply hydradx/model/amm/arbitrage_agent.py:6]: Made this change. The "text_" names are an artifact from before a recent refactor but I will just leave them consistent for now.

**Review by jepidoptera — APPROVED** [review]

**Pull request merged**